.vscode/launch.json

```diff
@@ -0,0 +1,177 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Safety Auth Login",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
+            "module": "safety",
+            "args": ["auth"],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Safety Auth Logout",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
+            "module": "safety",
+            "args": ["auth", "logout"],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Safety Scan",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
+            "module": "safety",
+            "args": ["scan"],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Safety Scan --detailed-output",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
+            "module": "safety",
+            "args": ["scan", "--detailed-output"],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Safety Scan --debug",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
+            "module": "safety",
+            "args": ["--debug","scan"],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Safety Scan --disable-optional-telemetry",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
+            "module": "safety",
+            "args": ["--disable-optional-telemetry","scan"],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Safety Scan --output json",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
+            "module": "safety",
+            "args": ["scan", "--output", "json", "--output-file", "test.json"],
+            "console": "integratedTerminal"
+        },
+        {
+            "name": "Safety Check",
+            "type": "debugpy",
+            "request": "launch",
+            "python": "/usr/local/bin/python",
+            "module": "safety",
```

Comment:
A lot of these configs are pretty similar besides small differences, wonder if there is a way we can abstract that to avoid the redundancies